package.json

```diff
@@ -13,28 +13,25 @@
   ],
   "types": "./lib/index.d.ts",
   "engines": {
-    "node": ">=8.12.0"
+    "node": ">=10.0"
   },
   "scripts": {
     "test": "jest",
-    "storybook": "run-s build:svgs:index storybook:dev",
+    "storybook": "run-s build:icons:index storybook:dev",
     "storybook:dev": "start-storybook --port 9001 --config-dir .storybook --static-dir ./public",
     "build-storybook": "build-storybook --config-dir .storybook --output-dir .out --static-dir ./public",
-    "clean": "rimraf dist lib src/icons/",
```

Comment:
CHANGELOG:

1. No longer clean `src/icons`--we're committing it now.
1. No longer build icons then use `create-index`--we're committing the icon setup, so we only build the index now.
1. Run `tsc` after `create-index` to ensure type compliance. 
1. Update `@svgr` packages.